protected/humhub/docs/guide/admin/installation-configuration.md

```diff
@@ -18,32 +18,43 @@ CronJobs and Asynchronous Job Processing (v1.3+)
 ---------------------------
 
 **CronJobs** are used to execute scheduled tasks as for example sending _summary emails_ or _search index optimization_
-and need to be configured in your server environment.
+and they need to be configured in your server environment.
 
-The scheduled job runner can be executed manually as follows:
+The scheduled job-runner can be executed manually as follows:
 
 ```
 > /usr/bin/php /path/to/humhub/protected/yii cron/run
 ```
 
-**Asynchronous Jobs** are used to execute potentially long running tasks in the background and can either be run by
-an extra CronJob or by other job-runner alternatives. Please see [Asynchronous Tasks](asynchronous-tasks.md) for more 
-details about queuing and job processing options.
+**Asynchronous Jobs** are used to execute potentially long running tasks in the background and can either be:
 
-The asynchronous job-runner can be executed manually as follows:
+- run by an extra CronJob (see below),
+- or by a job-runner alternative.
+
+> Please see [Asynchronous Tasks](asynchronous-tasks.md) for more details about queuing and job processing options.
+
+The asynchronous job-runner can also be executed manually as follows:
 
 ```
 > /usr/bin/php /path/to/humhub/protected/yii queue/run
 ```
 
 **Example CronTab configuration:**
 
+These Cronjobs can be runned together if you're not using any other job-runner (like Supervisor or Systemd):
+
 ```
-* * * * *  /usr/bin/php /path/to/humhub/protected/yii queue/run >/dev/null 2>&1
-* * * * *  /usr/bin/php /path/to/humhub/protected/yii cron/run >/dev/null 2>&1
+* * * * * /usr/bin/php /path/to/humhub/protected/yii queue/run >/dev/null 2>&1
+* * * * * /usr/bin/php /path/to/humhub/protected/yii cron/run >/dev/null 2>&1
 ```
 
-> Warning: You only require this cron jobs if there is no other job worker configured. See [Asynchronous Tasks](asynchronous-tasks.md) for more details.
+> Warning: These two cron jobs are only **both required** if there is no other job worker configured. See [Asynchronous Tasks](asynchronous-tasks.md) for more details. Please also note that aditionnal job-workers can be configured **only in dedicated environments, not shared hostings**.
```

Comment:
- would change it to "if there is no other worker configured" in the first sentence
- Please also note that **additional** job-workers ...